docs/root/intro/arch_overview/http/upgrades.rst

```diff
@@ -85,7 +85,7 @@ and forward the HTTP payload upstream. On receipt of initial TCP data from upstr
 will synthesize 200 response headers, and then forward the TCP data as the HTTP response body.
 
 .. warning::
-  This mode of CONNECT support can create major security holes if configured correctly, as the upstream
+  This mode of CONNECT support can create major security holes if not configured correctly, as the upstream
```

Comment:
I noticed this along the way, and assumed this was the intention.

Could have also possibly meant "... create holes _even if_ configured correctly ..."?